modules/data-entry/src/main/frontend/src/questionnaire/Question.jsx

```diff
@@ -38,7 +38,7 @@ function Question (props) {
       <CardHeader
         title={text}
         titleTypographyProps={{ variant: 'h6' }}
-        subheader={isEdit ? <MDEditor.Markdown className={classes.markdown} source={description} /> : null}
+        subheader={<MDEditor.Markdown className={classes.markdown} source={description} />}
```

Comment:
I wonder if it's a worthwhile optimization to only render this if there actually is a description.

```suggestion
        subheader={description ? <MDEditor.Markdown className={classes.markdown} source={description} /> : null}
```

Reply:
No need for it